html/elements/optgroup.json

```diff
@@ -21,13 +21,13 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": true
+              "version_added": "2"
```

Comment:
Why "2" and not "1" or "1.5"?

Reply:
I dug a bit: <optgroup> was supported in Netscape 6/7 so Firefox 1 did support it. So "1" here and "4" for Firefox for Android